### DISCUSSION TITLE
Where is the Pacman-like symbol?

### DISCUSSION THREAD

**Discussion by NNBnh** [ryanoasis/nerd-fonts#906] [Q&A] [answered]
I'm unable to find ![image](https://user-images.githubusercontent.com/43980777/187871725-86e164b1-9a4b-496d-b79a-15b3328b2cc3.png)...

I saw ![image](https://user-images.githubusercontent.com/43980777/187871725-86e164b1-9a4b-496d-b79a-15b3328b2cc3.png) on [Neofetch-themes](https://github.com/Chick2D/neofetch-themes#insert-name):

![img](https://camo.githubusercontent.com/6c7f540c6ec6d47376a1555bc85c93ace928b2ebffb606a872d88a28ec0f78ec/68747470733a2f2f63646e2e646973636f72646170702e636f6d2f6174746163686d656e74732f3838333133383433373431333232303431322f3933303437393130303233323039373834322f756e6b6e6f776e2e706e67)

But when I copy the ![image](https://user-images.githubusercontent.com/43980777/187871725-86e164b1-9a4b-496d-b79a-15b3328b2cc3.png) symbol, it show ![image](https://user-images.githubusercontent.com/43980777/187874756-fe3b654f-4753-4f8f-af87-697e5693ba03.png) on my terminal.

Then I try to search ![image](https://user-images.githubusercontent.com/43980777/187871725-86e164b1-9a4b-496d-b79a-15b3328b2cc3.png) on the [Cheat Sheet](https://www.nerdfonts.com/cheat-sheet), I even scroll through all icons and search it one by one but still can't find it :disappointed:

I've looking for ![image](https://user-images.githubusercontent.com/43980777/187871725-86e164b1-9a4b-496d-b79a-15b3328b2cc3.png) for 2 hours. Help me please...

**Comment by NNBnh** [comment, marked as answer]
Stupid me didn't notice:

> NOTE: On Arch-based distros, run `yay -S ttf-material-design-icons-extended`, otherwise the `pacman` packages won't work. On other distros you can install Material Design Icons Desktop TTF from [this repository](https://github.com/Templarian/MaterialDesign-Font).

Sorry for bothering...

**Reply by NNBnh** [reply]
You might also be interested in [this Unicode symbol](https://superuser.com/a/1701397).

**Reply by ChiefMikeK** [reply]
## Any Info on what font was used for this?

![legacyComputingSup](https://user-images.githubusercontent.com/20070053/189096005-2dcfacae-a0c1-41e6-8077-4b7d7c2b3015.jpg)
